### PR TITLE
Removing Jack, Adding Android 26

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

**Review by damienomurchu — APPROVED** [review]
Code fine & functionality tested and verified. Circle CI build failing on gradle dependencies however

Steps taken to verify:
- code visually inspected
- app built in studio and configured with an instance of welcome-cloud template on devexp.us.demos cluster
- app cloned & built locally, and all functionality tested on API 25 and Android-O emulators

File: app/build.gradle [hunk 1]
```diff
@@ -5,24 +5,16 @@ plugins {
...
         versionCode 1
         versionName "1.0"
 
-        jackOptions {
-            enabled true
-        }
-    }
-
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
```
Comment (danielpassos) [inline app/build.gradle]: Just wondering if we should not still using Java 8

https://developer.android.com/studio/write/java8-support.html#migrate

Comment (damienomurchu) [reply app/build.gradle]: Yes, seems we need to keep the referenced lines (& Jack) unless we upgrade the android gradle plugin to 3.0.0-alpha1 ? At the moment we're still using 2.3.3 (https://github.com/feedhenry-templates/welcome-android-gradle/blob/master/build.gradle#L8)

Comment (danielpassos) [reply app/build.gradle]: I think update to alpha should be part of the Android O update

**Review by danielpassos — APPROVED** [review]

**Comment by danielpassos** [comment]
Landed at Android-O branch https://github.com/feedhenry-templates/welcome-android-gradle/commit/0f115df8783b3f7d35adfe34b7864d123998fdb8

**Pull request closed without merging**